src/lib/dapp/index.ts

```diff
@@ -63,6 +68,7 @@ export class DAppConnector {
   supportedChains: string[] = []
 
   walletConnectClient: SignClient | undefined
+  walletConnectModal: WalletConnectModal
```

Comment:
Is there a reason why the modal instance should be a connector property?

Reply:
To support the connect functionality through modals. And also is exposed to be able to manage posible events. 
Like, ```walletConnectModal.subscribeModal```